.github/workflows/go.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           go-version: "1.20.0"
 
       - name: Install and build GUI dependency with yarn
-        run: yarn && yarn
+        run: yarn && yarn build
```

Comment:
Ideally I would rewrite history to `fixup` this commit (merging it with the previous one).